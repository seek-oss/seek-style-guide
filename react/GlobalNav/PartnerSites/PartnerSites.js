import styles from './PartnerSites.less';

import React from 'react';
import PropTypes from 'prop-types';

import ScreenReaderOnly from '../../ScreenReaderOnly/ScreenReaderOnly';

const linksObject = {
  AU: {
    jobsUrl: 'https://www.seek.com.au',
    learningUrl: 'https://www.seek.com.au/learning/?campaigncode=seek_banner_29&sc_trk=skj-courses-link',
    businessUrl: 'https://www.seekbusiness.com.au/?tracking=sk:main:au:nav:bus',
    volunteerUrl: 'https://www.volunteer.com.au/?tracking=SKMAU:main+header'
  },
  NZ: {
    jobsUrl: 'https://www.seek.co.nz',
    learningUrl: 'https://www.seeklearning.co.nz/?campaigncode=seek_banner_29&sc_trk=skj-courses-link',
    businessUrl: 'https://www.seekbusiness.com.au/?site=nz&tracking=sk:main:nz:nav:bus',
    volunteerUrl: 'https://seekvolunteer.co.nz/?tracking=SKMNZ:main+header'
  }
};

const err = () => {
  const e = new Error('Link key not found');
  e.name = 'PartnerSiteError';
  throw e;
};

const localeLens = links => locale => links ? links[locale] : err();
const linkLens = locale => link => typeof locale[link] === 'string' && locale[link] || err();

export default function PartnerSites({ locale, linkRenderer, activePartnerSite }) {
  const linkView = linkLens(localeLens(linksObject)(locale));

  const isJobsActive = activePartnerSite === 'jobs';
  const isCoursesActive = activePartnerSite === 'courses';
  const isBusinessesActive = activePartnerSite === 'businesses';
  const isVolunteeringActive = activePartnerSite === 'volunteering';

  return (
    <nav role="navigation" aria-labelledby="PartnerSites">

      <ScreenReaderOnly>
        <h1 id="PartnerSites">Partner Sites</h1>
      </ScreenReaderOnly>

      <ul className={styles.list}>
        <li>
          {isJobsActive ?
            <span className={`${styles.link} ${styles.link_isActive}`} title="SEEK Jobs">Jobs</span> :
            linkRenderer({
              'data-analytics': 'header:jobs',
              className: `${styles.link} ${styles.link_isJobs}`,
              href: linkView('jobsUrl'),
              title: 'SEEK Jobs',
              children: 'Jobs'
            })
          }
        </li>
        <li>
          {isCoursesActive ?
            <span className={`${styles.link} ${styles.link_isActive}`} title="SEEK Learning">Courses</span> :
            linkRenderer({
              'data-analytics': 'header:courses',
              className: `${styles.link} ${styles.link_isLearning}`,
              href: linkView('learningUrl'),
              title: 'SEEK Learning',
              children: 'Courses'
            })
          }
        </li>
        <li>
          {isBusinessesActive ?
            <span className={`${styles.link} ${styles.link_isActive}`} title="SEEK Business">Businesses for sale</span> :
            linkRenderer({
              'data-analytics': 'header:business+for+sale',
              className: `${styles.link} ${styles.link_isBusiness}`,
              href: linkView('businessUrl'),
              title: 'SEEK Business',
              children: 'Businesses for sale'
            })
          }
        </li>
        <li>
          {isVolunteeringActive ?
            <span className={`${styles.link} ${styles.link_isActive}`} title="SEEK Volunteer">Volunteering</span> :
            linkRenderer({
              'data-analytics': 'header:volunteering',
              className: `${styles.link} ${styles.link_isVolunteering}`,
              href: linkView('volunteerUrl'),
              title: 'SEEK Volunteer',
              children: 'Volunteering'
            })
          }
        </li>
      </ul>
    </nav>
  );
}

PartnerSites.propTypes = {
  locale: PropTypes.oneOf(['AU', 'NZ']).isRequired,
  linkRenderer: PropTypes.func.isRequired,
  activePartnerSite: PropTypes.oneOf(['jobs', 'courses', 'businesses', 'volunteering']).isRequired
};
