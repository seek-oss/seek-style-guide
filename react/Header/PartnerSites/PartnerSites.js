import styles from './PartnerSites.less';

import React, { PropTypes } from 'react';
import ScreenReaderOnly from '../../Accessibility/ScreenReaderOnly';

const linksObject = {
  AU: {
    learningUrl: 'https://www.seeklearning.com.au/?campaigncode=seek_banner_29&sc_trk=skj-courses-link',
    businessUrl: 'https://www.seekbusiness.com.au/?tracking=sk:main:au:nav:bus',
    volunteerUrl: 'https://www.volunteer.com.au/?tracking=SKMAU:main+header'
  },
  NZ: {
    learningUrl: 'https://www.seeklearning.co.nz/?campaigncode=seek_banner_29&sc_trk=skj-courses-link',
    businessUrl: 'https://www.seekbusiness.com.au/?site=nz&tracking=sk:main:nz:nav:bus',
    volunteerUrl: 'https://seekvolunteer.co.nz/?tracking=SKMNZ:main+header'
  }
};

const err = () => {
  const e = new Error('Link key not found');
  e.name = 'PartnerSiteError';
  throw e;
};

const localeLens = links => locale => links ? links[locale] : err();
const linkLens = locale => link => typeof locale[link] === 'string' && locale[link] || err();

export default function PartnerSites({ locale, linkRenderer }) {
  const linkView = linkLens(localeLens(linksObject)(locale));

  return (
    <nav role="navigation" aria-labelledby="PartnerSites">

      <ScreenReaderOnly>
        <h1 id="PartnerSites">Partner Sites</h1>
      </ScreenReaderOnly>

      <ul className={styles.list}>
        <li>
          <span className={`${styles.link} ${styles.link_isJobs}`} title="SEEK Jobs">Jobs</span>
        </li>
        <li>
          {
            linkRenderer({
              'data-analytics': 'header:courses',
              className: `${styles.link} ${styles.link_isLearning}`,
              href: linkView('learningUrl'),
              title: 'SEEK Learning',
              children: 'Courses'
            })
          }
        </li>
        <li>
          {
            linkRenderer({
              'data-analytics': 'header:business+for+sale',
              className: `${styles.link} ${styles.link_isBusiness}`,
              href: linkView('businessUrl'),
              title: 'SEEK Business',
              children: 'Businesses for sale'
            })
          }
        </li>
        <li>
          {
            linkRenderer({
              'data-analytics': 'header:volunteering',
              className: `${styles.link} ${styles.link_isVolunteering}`,
              href: linkView('volunteerUrl'),
              title: 'SEEK Volunteer',
              children: 'Volunteering'
            })
          }
        </li>
      </ul>
    </nav>
  );
}

PartnerSites.propTypes = {
  locale: PropTypes.string.isRequired,
  linkRenderer: PropTypes.func.isRequired
};
