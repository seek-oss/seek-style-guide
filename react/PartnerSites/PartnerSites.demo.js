import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import {
  PartnerSites,
  Hidden,
  PageBlock,
  Section,
  Text
} from 'seek-style-guide/react';

const Container = ({ component: DemoComponent, componentProps }) => (
  <Fragment>
    <DemoComponent {...componentProps} />
    <Hidden desktop>
      <PageBlock>
        <Section>
          <Text strong>This component is not visible on mobile.</Text>
        </Section>
      </PageBlock>
    </Hidden>
  </Fragment>
);

Container.propTypes = {
  component: PropTypes.any,
  componentProps: PropTypes.object.isRequired
};

const ROUTE = '/partner-sites';

export default {
  route: ROUTE,
  title: 'Partner Sites',
  category: 'Layout',
  component: PartnerSites,
  container: Container,
  options: [
    {
      label: 'Active Product',
      type: 'radio',
      states: [
        null,
        'Jobs',
        'Courses',
        'Businesses for sale',
        'Volunteering'
      ].map(activeSite => ({
        label: activeSite || 'No active site',
        transformProps: props => ({
          ...props,
          activeSite
        })
      }))
    },
    {
      label: 'Locale',
      type: 'radio',
      states: [
        {
          label: 'AU',
          transformProps: props => props
        },
        {
          label: 'NZ',
          transformProps: props => ({
            ...props,
            locale: 'NZ'
          })
        }
      ]
    },
    {
      label: 'Override Locale Links',
      type: 'checkbox',
      states: [
        {
          label: 'Override Locale Links',
          transformProps: props => ({
            ...props,
            localeLinks: {
              AU: {
                'data-analytics': 'header:au+employer',
                href: 'https://talent.seek.com.au',
                title: 'SEEK Employer Australia'
              },
              NZ: {
                'data-analytics': 'header:nz+employer',
                href: 'https://talent.seek.co.nz',
                title: 'SEEK Employer New Zealand'
              }
            }
          })
        }
      ]
    }
  ]
};
