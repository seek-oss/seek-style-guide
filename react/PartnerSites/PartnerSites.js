import styles from './PartnerSites.less';
import React from 'react';
import PropTypes from 'prop-types';

import Hidden from '../Hidden/Hidden';
import PageBlock from '../PageBlock/PageBlock';
import Locales from './Locales/Locales';
import Sites from './Sites/Sites';

const defaultLinkRenderer = props => <a {...props} />;

export default function PartnerSites({
  locale,
  linkRenderer,
  activeSite,
  localeLinks
}) {
  return (
    <Hidden component={PageBlock} print mobile className={styles.root}>
      <div className={styles.content}>
        <Sites
          locale={locale}
          linkRenderer={linkRenderer}
          activeSite={activeSite}
        />
        <div className={styles.locale}>
          <Locales
            locale={locale}
            linkRenderer={linkRenderer}
            localeLinks={localeLinks}
          />
        </div>
      </div>
    </Hidden>
  );
}

PartnerSites.displayName = 'PartnerSites';

const localeLink = PropTypes.shape({
  'data-analytics': PropTypes.string,
  href: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired
});

PartnerSites.propTypes = {
  locale: PropTypes.oneOf(['AU', 'NZ']),
  linkRenderer: PropTypes.func,
  activeSite: PropTypes.oneOf([
    'Jobs',
    'Courses',
    'Businesses for sale',
    'Volunteering'
  ]),
  localeLinks: PropTypes.shape({
    AU: localeLink.isRequired,
    NZ: localeLink.isRequired
  })
};

PartnerSites.defaultProps = {
  locale: 'AU',
  linkRenderer: defaultLinkRenderer,
  activeSite: null,
  localeLinks: {
    AU: {
      'data-analytics': 'header:au+homepage',
      href: 'https://www.seek.com.au',
      title: 'SEEK Australia'
    },
    NZ: {
      'data-analytics': 'header:nz+homepage',
      href: 'https://www.seek.co.nz',
      title: 'SEEK New Zealand'
    }
  }
};
