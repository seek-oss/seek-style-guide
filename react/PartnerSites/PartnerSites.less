@import (reference) "~seek-style-guide/theme";

.root {
  width: 100%;
  padding: 0 (@gutter-width / 2);
  background-color: @sk-background;
}

.content {
  position: relative;
  display: flex;
  flex-grow: 1;
  justify-content: center;
}

.locale {
  position: absolute;
  right: 0;
}
