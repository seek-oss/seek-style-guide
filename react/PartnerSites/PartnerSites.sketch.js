import React from 'react';
import PropTypes from 'prop-types';
import PartnerSites from './PartnerSites';

const DummyLinkRenderer = ({ href, ...props }) => <a {...props} />;

DummyLinkRenderer.propTypes = {
  href: PropTypes.string
};

export const blockSymbols = {
  'PartnerSites/Australia/No Active Product': (
    <PartnerSites linkRenderer={DummyLinkRenderer} />
  ),
  'PartnerSites/Australia/Jobs Product Active': (
    <PartnerSites linkRenderer={DummyLinkRenderer} activeSite="Jobs" />
  ),
  'PartnerSites/Australia/Courses Product Active': (
    <PartnerSites linkRenderer={DummyLinkRenderer} activeSite="Courses" />
  ),
  'PartnerSites/Australia/BusinessesForSale Product Active': (
    <PartnerSites
      linkRenderer={DummyLinkRenderer}
      activeSite="Businesses for sale"
    />
  ),
  'PartnerSites/Australia/Volunteering Product Active': (
    <PartnerSites linkRenderer={DummyLinkRenderer} activeSite="Volunteering" />
  ),
  'PartnerSites/New Zealand/No Active Product': (
    <PartnerSites locale="NZ" linkRenderer={DummyLinkRenderer} />
  ),
  'PartnerSites/New Zealand/Jobs Product Active': (
    <PartnerSites
      locale="NZ"
      linkRenderer={DummyLinkRenderer}
      activeSite="Jobs"
    />
  ),
  'PartnerSites/New Zealand/Courses Product Active': (
    <PartnerSites
      locale="NZ"
      linkRenderer={DummyLinkRenderer}
      activeSite="Courses"
    />
  ),
  'PartnerSites/New Zealand/BusinessesForSale Product Active': (
    <PartnerSites
      locale="NZ"
      linkRenderer={DummyLinkRenderer}
      activeSite="Businesses for sale"
    />
  ),
  'PartnerSites/New Zealand/Volunteering Product Active': (
    <PartnerSites
      locale="NZ"
      linkRenderer={DummyLinkRenderer}
      activeSite="Volunteering"
    />
  )
};
