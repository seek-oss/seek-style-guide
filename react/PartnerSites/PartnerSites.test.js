import React from 'react';
import { render } from 'enzyme';

import PartnerSites from './PartnerSites';

const renderPartnerSites = props => render(<PartnerSites {...props} />);

describe('Header:', () => {
  const links = {
    AU: {
      'data-analytics': 'header:au+employer',
      href: 'https://talent.seek.com.au',
      title: 'SEEK Employer Australia',
      children: 'AU'
    },
    NZ: {
      'data-analytics': 'header:nz+employer',
      href: 'https://talent.seek.co.nz',
      title: 'SEEK Employer New Zealand',
      children: 'NZ'
    }
  };
  it('should render with locale of AU', () => {
    expect(renderPartnerSites({ locale: 'AU' })).toMatchSnapshot();
  });

  it('should render with locale of NZ', () => {
    expect(renderPartnerSites({ locale: 'NZ' })).toMatchSnapshot();
  });

  it('should render with overridden AU Locale link', () => {
    expect(
      renderPartnerSites({ locale: 'NZ', localeLinks: links })
    ).toMatchSnapshot();
  });

  it('should render with overridden NZ Locale link', () => {
    expect(
      renderPartnerSites({ locale: 'AU', localeLinks: links })
    ).toMatchSnapshot();
  });
});
