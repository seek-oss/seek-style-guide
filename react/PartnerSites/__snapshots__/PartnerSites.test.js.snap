// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header: should render with locale of AU 1`] = `
<div
  class="PartnerSites__root Hidden__mobile Hidden__print"
>
  <div
    class="PageBlock__content"
  >
    <div
      class="PartnerSites__content"
    >
      <nav
        aria-labelledby="Sites"
        role="navigation"
      >
        <span
          class="ScreenReaderOnly__root"
        >
          <h1
            id="Sites"
          >
            SEEK Sites
          </h1>
        </span>
        <ul
          class="Sites__list"
        >
          <li>
            <a
              class="Sites__link Sites__link_isJobs"
              data-analytics="header:jobs"
              href="https://www.seek.com.au"
              title="SEEK Jobs"
            >
              Jobs
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isLearning"
              data-analytics="header:courses"
              href="https://www.seek.com.au/learning/?campaigncode=seek_banner_29&sc_trk=skj-courses-link"
              title="SEEK Learning"
            >
              Courses
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isBusiness"
              data-analytics="header:business+for+sale"
              href="https://www.seekbusiness.com.au/?tracking=sk:main:au:nav:bus"
              title="SEEK Business"
            >
              Businesses for sale
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isVolunteering"
              data-analytics="header:volunteering"
              href="https://www.volunteer.com.au/?tracking=SKMAU:main+header"
              title="SEEK Volunteer"
            >
              Volunteering
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="PartnerSites__locale"
      >
        <nav
          aria-labelledby="Locales"
          role="navigation"
        >
          <span
            class="ScreenReaderOnly__root"
          >
            <h1
              id="Locales"
            >
              Select your country
            </h1>
          </span>
          <ul
            class="Locales__list"
          >
            <li>
              <span
                class="Locales__locale_isActive"
              >
                AU
              </span>
            </li>
            <li>
              <a
                class="Locales__localeLink"
                data-analytics="header:nz+homepage"
                href="https://www.seek.co.nz"
                title="SEEK New Zealand"
              >
                NZ
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</div>
`;

exports[`Header: should render with locale of NZ 1`] = `
<div
  class="PartnerSites__root Hidden__mobile Hidden__print"
>
  <div
    class="PageBlock__content"
  >
    <div
      class="PartnerSites__content"
    >
      <nav
        aria-labelledby="Sites"
        role="navigation"
      >
        <span
          class="ScreenReaderOnly__root"
        >
          <h1
            id="Sites"
          >
            SEEK Sites
          </h1>
        </span>
        <ul
          class="Sites__list"
        >
          <li>
            <a
              class="Sites__link Sites__link_isJobs"
              data-analytics="header:jobs"
              href="https://www.seek.co.nz"
              title="SEEK Jobs"
            >
              Jobs
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isLearning"
              data-analytics="header:courses"
              href="https://www.seeklearning.co.nz/?campaigncode=seek_banner_29&sc_trk=skj-courses-link"
              title="SEEK Learning"
            >
              Courses
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isBusiness"
              data-analytics="header:business+for+sale"
              href="https://www.seekbusiness.com.au/?site=nz&tracking=sk:main:nz:nav:bus"
              title="SEEK Business"
            >
              Businesses for sale
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isVolunteering"
              data-analytics="header:volunteering"
              href="https://seekvolunteer.co.nz/?tracking=SKMNZ:main+header"
              title="SEEK Volunteer"
            >
              Volunteering
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="PartnerSites__locale"
      >
        <nav
          aria-labelledby="Locales"
          role="navigation"
        >
          <span
            class="ScreenReaderOnly__root"
          >
            <h1
              id="Locales"
            >
              Select your country
            </h1>
          </span>
          <ul
            class="Locales__list Locales__list_isNZ"
          >
            <li>
              <a
                class="Locales__localeLink"
                data-analytics="header:au+homepage"
                href="https://www.seek.com.au"
                title="SEEK Australia"
              >
                AU
              </a>
            </li>
            <li>
              <span
                class="Locales__locale_isActive"
              >
                NZ
              </span>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</div>
`;

exports[`Header: should render with overridden AU Locale link 1`] = `
<div
  class="PartnerSites__root Hidden__mobile Hidden__print"
>
  <div
    class="PageBlock__content"
  >
    <div
      class="PartnerSites__content"
    >
      <nav
        aria-labelledby="Sites"
        role="navigation"
      >
        <span
          class="ScreenReaderOnly__root"
        >
          <h1
            id="Sites"
          >
            SEEK Sites
          </h1>
        </span>
        <ul
          class="Sites__list"
        >
          <li>
            <a
              class="Sites__link Sites__link_isJobs"
              data-analytics="header:jobs"
              href="https://www.seek.co.nz"
              title="SEEK Jobs"
            >
              Jobs
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isLearning"
              data-analytics="header:courses"
              href="https://www.seeklearning.co.nz/?campaigncode=seek_banner_29&sc_trk=skj-courses-link"
              title="SEEK Learning"
            >
              Courses
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isBusiness"
              data-analytics="header:business+for+sale"
              href="https://www.seekbusiness.com.au/?site=nz&tracking=sk:main:nz:nav:bus"
              title="SEEK Business"
            >
              Businesses for sale
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isVolunteering"
              data-analytics="header:volunteering"
              href="https://seekvolunteer.co.nz/?tracking=SKMNZ:main+header"
              title="SEEK Volunteer"
            >
              Volunteering
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="PartnerSites__locale"
      >
        <nav
          aria-labelledby="Locales"
          role="navigation"
        >
          <span
            class="ScreenReaderOnly__root"
          >
            <h1
              id="Locales"
            >
              Select your country
            </h1>
          </span>
          <ul
            class="Locales__list Locales__list_isNZ"
          >
            <li>
              <a
                class="Locales__localeLink"
                data-analytics="header:au+employer"
                href="https://talent.seek.com.au"
                title="SEEK Employer Australia"
              >
                AU
              </a>
            </li>
            <li>
              <span
                class="Locales__locale_isActive"
              >
                NZ
              </span>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</div>
`;

exports[`Header: should render with overridden NZ Locale link 1`] = `
<div
  class="PartnerSites__root Hidden__mobile Hidden__print"
>
  <div
    class="PageBlock__content"
  >
    <div
      class="PartnerSites__content"
    >
      <nav
        aria-labelledby="Sites"
        role="navigation"
      >
        <span
          class="ScreenReaderOnly__root"
        >
          <h1
            id="Sites"
          >
            SEEK Sites
          </h1>
        </span>
        <ul
          class="Sites__list"
        >
          <li>
            <a
              class="Sites__link Sites__link_isJobs"
              data-analytics="header:jobs"
              href="https://www.seek.com.au"
              title="SEEK Jobs"
            >
              Jobs
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isLearning"
              data-analytics="header:courses"
              href="https://www.seek.com.au/learning/?campaigncode=seek_banner_29&sc_trk=skj-courses-link"
              title="SEEK Learning"
            >
              Courses
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isBusiness"
              data-analytics="header:business+for+sale"
              href="https://www.seekbusiness.com.au/?tracking=sk:main:au:nav:bus"
              title="SEEK Business"
            >
              Businesses for sale
            </a>
          </li>
          <li>
            <a
              class="Sites__link Sites__link_isVolunteering"
              data-analytics="header:volunteering"
              href="https://www.volunteer.com.au/?tracking=SKMAU:main+header"
              title="SEEK Volunteer"
            >
              Volunteering
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="PartnerSites__locale"
      >
        <nav
          aria-labelledby="Locales"
          role="navigation"
        >
          <span
            class="ScreenReaderOnly__root"
          >
            <h1
              id="Locales"
            >
              Select your country
            </h1>
          </span>
          <ul
            class="Locales__list"
          >
            <li>
              <span
                class="Locales__locale_isActive"
              >
                AU
              </span>
            </li>
            <li>
              <a
                class="Locales__localeLink"
                data-analytics="header:nz+employer"
                href="https://talent.seek.co.nz"
                title="SEEK Employer New Zealand"
              >
                NZ
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</div>
`;
